admin-app/src/components/GameList.tsx

```diff
@@ -15,3 +22,14 @@ export const GameList = (props: GameListProps) => (
     </Datagrid>
   </List>
 );
+
+const gameFilters = [
+  <SelectInput
+    source="status"
+    choices={[
+      { id: 'INITIALIZED', name: 'Initialized' },
+      { id: 'RUNNING', name: 'Running' },
+      { id: 'ENDED', name: 'Ended' },
```

Comment:
Just for my information, did you have to add these values cause RA has no way of guessing them, or because the uppercase labels bothered you?

Reply:
RA can scrap the values inside the displayed lists of items but not guess other values. So I prefer be explicit define all the accepted values manually.  Ideally we would export the type from the nestjs entity(or extract it into a types file) but that's for another day !